appcues/src/main/java/com/appcues/debugger/screencapture/AndroidTargetingStrategy.kt

```diff
@@ -153,11 +154,12 @@ private fun View.asCaptureView(screenBounds: Rect): ViewElement? {
             val semanticsOwner = semanticsOwnerField.get(this) as SemanticsOwner
             val composeChildren = listOf(semanticsOwner.rootSemanticsNode.asCaptureView(context))
             children.addAll(composeChildren)
-        } catch (_: Exception) {
+        } catch (ex: Exception) {
             // Catching and swallowing exceptions here with the Compose view handling in case
             // something changes in the future that breaks the expected structure being accessed
             // through reflection here. If anything goes wrong within this block, prefer to continue
             // processing the remainder of the view tree as best we can.
+            Log.e("Appcues", "error processing Compose layout, ${ex.message}")
```

Comment:
There is really not a clear access to Logcues at this point, and the targeting strategy is kept in a static spot in `Appcues.elementTargeting`. We could add a one-off log message here to provide some feedback in the case of an error like what we've seen recently. I can also remove this if undesired to use Logcat directly like this, outside of our normal logging structure.

Reply:
i see no other way to log possible erros here, unless we make the strategy non-static